src/runtime/components/ContentDoc.vue

```diff
@@ -94,12 +97,12 @@ export default defineComponent({
         // Default slot
         default: slots?.default
           ? ({ data, refresh, isPartial }: any) => {
-              if (head) { useContentHead(data) }
+              if (head && enableContentHead) { useContentHead(data) }
```

Comment:
I think it is a better approach if we use `enableContentHead` as default value `head` props and let users overwrite it in special cases if they want to.

Reply:
I don't understand this comment. Could you explain `we use enableContentHead as default value head props`.

Reply:
This component accepts a `head` props with `true` as default, we can set the default to `undefined` then assing `enableContentHead` to it if it's undefined

maybe this snippet describes it better
```ts
const shouldInjectContentHead = props.head === undefined ? enableContentHead : props.head

if (shouldInjectContentHead) { useContentHead(data) }

```

Reply:
Okkk, I understand. Will update the PR.